### ISSUE TITLE
Exclude all dart_code_metrics rules at once

### ISSUE THREAD

**Issue by PiN73** [dart-code-checker/dart-code-metrics#286]
#258 allows to exclude rules:
```yaml
dart_code_metrics:
  rules:
    no_equal_arguments:
      #258
      exclude:
        - test/**
```

This is useful to exclude generated files (e.g. [freezed](https://pub.dev/packages/freezed)). But then it will be duplicated:

```yaml
dart_code_metrics:
  rules:
    rule-1:
      # Ctrl+C
      exclude:
        - lib/**.freezed.dart
    rule-2:
      # Ctrl+V
      exclude:
        - lib/**.freezed.dart
    rule-3:
      # Ctrl+V
      exclude:
        - lib/**.freezed.dart
```

I think it would be good to have root exclude option in dart_code_metrics section:
```yaml
dart_code_metrics:
  # proposal
  exclude:
    - test/**
  rules:
    rule-1
    rule-2
    rule-3
```

Note: exclude in analyzer section would disable analysis for the files at all, even errors 
```yaml
analyzer:
  exclude:
    - lib/**/*.freezed.dart
```

**Are you willing to submit a pull request to implement this change?**
Yes, I want to try it

**Comment by PiN73** [comment]
There exists same issue for default Dart linter: https://github.com/dart-lang/linter/issues/320

But as dart_code_checker already supports per-rule exclude options, I think adding general one makes sense too

**Comment by incendial** [comment]
Available in 4.3.0 version.
